lib/kb_orthofinder/kb_orthofinderImpl.py

```diff
@@ -416,6 +427,10 @@ def annotate_plant_transcripts(self, ctx, input):
                     transcript_id = header
                     transcript_description = None
 
+                # anticipating if OF changed characters in ids, see above
+                if(transcript_id in alt_seq_ids_dict):
```

Comment:
no brackets needed

Reply:
If you run the code through a formatter like black, it might automatically get rid of this kind of thing. Another option would be using Ruff and trying the 'autofix' option.

Reply:
Fixed a lot of these in 9ca841d60089414785b076375ae088febb9d0668